src/population.py

```diff
@@ -0,0 +1,58 @@
+import random
+
+class Population():
+    def __init__(self, solutions, fitness):
+        self.solutions = solutions
+        self.fitness = fitness
+
+    def initial_population(self, population_size, test_size):
```

Comment:
Please add a default value for `test_size = 1`, for now. 

We need a more sophisticated way of marking test types (not only test_size). But for now, it is okay...